doc/source/whatsnew/v0.16.1.txt

```diff
@@ -27,7 +27,7 @@ Enhancements
 API changes
 ~~~~~~~~~~~
 
-
+- When subclassing ``DataFrame``, the correct class is returned, as specified in ``_constructor_sliced`` (PR 9632)
```

Comment:
Thanks, I think only referring to #9632 is enough (as 2859 has been closed already).